camel-idea-plugin/src/main/java/org/apache/camel/idea/annotator/CamelAnnotator.java

```diff
@@ -0,0 +1,220 @@
+/**
```

Comment:
I would consider renaming this to CamelEndpointAnnotator. We can in the future have other annotators for other kinds, like simple, xpath, data formats, and whatelse there comes down the road.